config/routes.rb

```diff
@@ -5,7 +5,13 @@
   # Can be used by load balancers and uptime monitors to verify that the app is live.
   get "up" => "rails/health#show", as: :rails_health_check
 
-  resources :notes, only: [:index, :show, :create, :update]
+  resources :notes, only: [:index, :show, :create, :update] do
+    member do
+      get 'beautiful_transcript'
+    end
+  end
 
   devise_for :users, controllers: { sessions: "sessions", registrations: "registrations" }
+  
+  get "/generate", to: "notes#create"
```

Comment:
we can’t do a POST from an extension apparently, so this is a get route that sends you to `notes#create`